Source/SIMPLib/Geometry/CompositeTransform.cpp

```diff
@@ -0,0 +1,160 @@
+/* ============================================================================
+* Copyright (c) 2009-2016 BlueQuartz Software, LLC
```

Comment:
Update the copyright year also. (In all new files)

Reply:
done